internal/service/todo/fw_data_source_mongodbatlas_atlas_users.go

```diff
@@ -1,4 +1,4 @@
-package mongodbatlas
+package todo
```

Comment:
is that `todo` because you still haven't figured a place?

Reply:
i wanted to try to change as few files as possible and focus on alert_configuration as an example. If we like it we would extend the same idea for resources in "todo" (the ones in the new FW) and eventually for all resources.

Reply:
Is the intention to merge this PR with todo package? Understand it will be temporary but to avoid leaving the `todo` package we can place the files directly in `internal/service`.

Reply:
it'll be merged with the real resource packages